modules/scale-down-data-plane.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="scale-down-data-plane_{context}"]
+= Scaling down the data plane to zero
+
+If you are not using the hosted control plane, to save the resources and cost you can scale down a data plane to zero.
+
+[NOTE]
+====
+Ensure you are prepared to scale down the data plane to zero. Because the workload from the worker nodes disappears after scaling down.
+====
+
+.Procedure
+
+. Set the `kubeconfig` file to access the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=<install_directory>/auth/kubeconfig
+----
+
+. Get the name of the `NodePool` resource associated to your hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodepool --namespace <HOSTED_CLUSTER_NAMESPACE>
+----
+
+. Optional: To prevent the pods from draining, add the `nodeDrainTimeout` field in the `NodePool` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit NodePool <nodepool> -o yaml --namespace <HOSTED_CLUSTER_NAMESPACE>
+----
++
+.Example output
+[source,terminal]
```

Comment:
Should this be [source, yaml]

additionally, I am used to seeing example resource output with a little more context at the beginning. LIke:
`apiVersion: v1alpha1
kind: NodePool
metadata:
  name: cluster`

Reply:
Like Step 3 here:
https://docs.openshift.com/container-platform/4.13/scalability_and_performance/using-node-tuning-operator.html#node-tuning-hosted-cluster_node-tuning-operator

See how there's the API context at the top?

Reply:
Thanks! Addressed